catalog/app/utils/error.ts

```diff
@@ -40,3 +40,16 @@ export class ErrorDisplay extends BaseError {
     super(headline, { headline, detail, object })
   }
 }
+
+export class ErrorCredentials extends BaseError {
```

Comment:
i don't think this util module is a proper place for this error class, probably `utils/AWS` or `utils/AWS/Credentials` is better. also, `CredentialsError` seems like a better name.

Reply:
It seemed to me as a module to rule all errors. If it's not then I can move it to AWS/Credentials

Reply:
I understand that CredentialsError is nicer but naming things from general to specific is more convenient from my experience: listing is better, search is faster. Vue styleguide section is related https://v2.vuejs.org/v2/style-guide/?redirect=true#Order-of-words-in-component-names-strongly-recommended

Reply:
> I understand that CredentialsError is nicer but naming things from general to specific is more convenient

yeah i see what you mean and i do that myself most of the times (i guess) as well

> It seemed to me as a module to rule all errors

it's more like a place for reusable general stuff, and more specific things should be kept closer to their domain